pkg/storage/filesystem.go

```diff
@@ -21,23 +21,17 @@ type Mountpoint struct {
 }
 
 // GetSupportedFilesystems returns the supported file systems for block devices,
-func GetSupportedFilesystems() ([]string, error) {
-	fd, err := os.Open("/proc/filesystems")
-	if err != nil {
-		return nil, err
+func GetSupportedFilesystems() (fstypes []string, err error) {
+	var bytes []byte
+	if bytes, err = ioutil.ReadFile("/proc/filesystems"); err != nil{
+		return nil, fmt.Errorf("Failed to read /proc/filesystems: %v", err)
 	}
-	filesystems := make([]string, 0)
-	scanner := bufio.NewScanner(fd)
-	for scanner.Scan() {
-		fields := strings.Split(scanner.Text(), "\t")
-		if fields[0] == "" {
-			filesystems = append(filesystems, fields[1])
+	for _, line := range strings.Split(string(bytes), "\n") {
+		if fields := strings.Fields(line); len(fields) == 1 {
```

Comment:
Please comment about the format (i.e. why len(fields) == 1)

// Two possibilities for lines in this file
// ext4
// nodev ext4
//
// nodev file systems are fs's which cannot be mounted through devices, so we don't want to list then here.